### ISSUE TITLE
Heroes list

### ISSUE THREAD

**Issue by toinouu** [SunDwarf/OWAPI#107]
Hi all ! 

it's not really an issue but more feature.

is it possible to have the list of heroes  => https://playoverwatch.com/en-us/heroes ?

thx !

**Comment by mikevb1** [comment]
There's no "hero list", but see the keys in either `[REGION][heroes][playtime][MODE]` or `[REGION][heroes][stats][MODE]`

**Comment by toinouu** [comment]
in fact I imagined this 

https://owapi.net/api/v2/heroes

```
"heroes": {
    "tank": {
        "reinhart",
        "winston",
        ......
    },
     "offense": {
        "genji",
        "phara",
        ......
    },
    ....
}
```
and maybe 

https://owapi.net/api/v2/heros/ana

```
"ana":  {
    "role": "suport",
    "difficulty": 3,
    "abilities" : { ... }
    ....
}
```

**Comment by SunDwarf** [comment]
Well, it'd be a v3 endpoint only.

PRs welcome, but if nobody has done it by the time I get back I will consider it.

**Comment by SunDwarf** [comment]
Closed by #111
